tuned_lens/scripts/ingredients.py

```diff
@@ -88,6 +88,9 @@ class Model:
     name: str
     """Name of model to use in the Huggingface Hub."""
 
+    int8: bool = field(action="store_true", default=False)
```

Comment:
It might be good to add a section to the lens training and evaluation tutorial on how to use this its in `docs/source/tutorials`.

Reply:
done